pom.xml

```diff
@@ -266,15 +266,15 @@
                                 <!-- Oracle image - we aim to use same version as Dev Services for Oracle so that we only download one image -->
                                 <oracle.image>docker.io/gvenzl/oracle-free:23-slim-faststart</oracle.image>
                                 <db2.image>docker.io/ibmcom/db2:11.5.8.0</db2.image>
-                                <mongodb.image>docker.io/library/mongo:5.0</mongodb.image>
+                                <mongodb.image>docker.io/library/mongo:7.0</mongodb.image>
                                 <redis.image>docker.io/library/redis:7.2</redis.image>
                                 <wiremock.image>docker.io/wiremock/wiremock:3.3.1</wiremock.image>
                                 <!-- TODO use official image when/if this fixed https://github.com/hashicorp/docker-consul/issues/184 -->
                                 <consul.image>docker.io/bitnami/consul:1.17.2</consul.image>
                                 <infinispan.image>quay.io/infinispan/server:14.0</infinispan.image>
                                 <infinispan.expected-log>Infinispan Server.*started in</infinispan.expected-log>
                                 <spring.cloud.server.image>quay.io/quarkusqeteam/spring-cloud-config-server:4.1</spring.cloud.server.image>
-                                <keycloak.image>quay.io/keycloak/keycloak:24.0</keycloak.image>
+                                <keycloak.image>quay.io/keycloak/keycloak:25.0</keycloak.image>
```

Comment:
Keycloak was bumped with https://github.com/quarkus-qe/quarkus-test-suite/pull/1900/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R277 as that framework version contain the keycloak 25 and causing some problem

Reply:
Rebased on that change, thank for pointing out!